lib/matplotlib/tests/test_patches.py

```diff
@@ -808,3 +808,32 @@ def test_default_capstyle():
 def test_default_joinstyle():
     patch = Patch()
     assert patch.get_joinstyle() == 'miter'
+
+
+@image_comparison(["autoscale_arc"], extensions=['png', 'svg'],
+                  style="mpl20")
+def test_autoscale_arc():
+    fig, axs = plt.subplots(1, 3, squeeze=False, figsize=(4, 1))
+    arc_lists = (
+        [Arc((0, 0), 1, 1, theta1=0, theta2=90)],
+        [Arc((0.5, 0.5), 1.5, 0.5, theta1=10, theta2=20)],
+        [Arc((0.5, 0.5), 1.5, 0.5, theta1=10, theta2=20),
+         Arc((0.5, 0.5), 2.5, 0.5, theta1=110, theta2=120),
+         Arc((0.5, 0.5), 3.5, 0.5, theta1=210, theta2=220),
+         Arc((0.5, 0.5), 4.5, 0.5, theta1=310, theta2=320)])
+
+    for ax, arcs in zip(axs[0], arc_lists):
+        for arc in arcs:
+            ax.add_patch(arc)
+        ax.autoscale()
+        ax.set_axis_off()
```

Comment:
removing the text makes sense, but I'm not sure removing the axes makes it easy to see what is going on here.  For instance, I'm not clear that you have tested any of your "stretch" logic here.

Reply:
I sort of agree that keeping the axes would be good for that purpose. I'll change that (despite the slightly larger file size).

The stretch thing happens when the "base" is an ellipsis (as opposed to a circle), so that should be tested here and also in the collection test.